CMakeLists.txt

```diff
@@ -1,11 +1,8 @@
 cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
 
 
-find_package(IRODS 4.2.7 EXACT REQUIRED)
-set(IRODS_PLUGIN_VERSION_MAJOR 2)
-set(IRODS_PLUGIN_VERSION_MINOR 7)
-set(IRODS_PLUGIN_VERSION_PATCH 0)
-set(IRODS_PLUGIN_VERSION "${IRODS_PLUGIN_VERSION_MAJOR}.${IRODS_PLUGIN_VERSION_MINOR}.${IRODS_PLUGIN_VERSION_PATCH}")
+find_package(IRODS 4.2.8 EXACT REQUIRED)
+set(IRODS_PLUGIN_VERSION "${IRODS_VERSION_MAJOR}.${IRODS_VERSION_MINOR}.${IRODS_VERSION_PATCH}.0")
```

Comment:
we're using 'IRODS_PLUGIN_REVISION' and then appending that variable to IRODS_VERSION...

please see 

https://github.com/irods/irods_resource_plugin_s3/blob/ce038716af667400e06d2293256ee54e19cf9584/CMakeLists.txt#L3-L4

and

https://github.com/irods/irods_resource_plugin_s3/blob/ce038716af667400e06d2293256ee54e19cf9584/CMakeLists.txt#L37